apps/backoffice/src/components/app-preview/index.tsx

```diff
@@ -0,0 +1,84 @@
+import { Close, Info, InfoOutlined, Title } from "@mui/icons-material";
+import {
+  Box,
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  FormControl,
+  InputLabel,
+  MenuItem,
+  Paper,
+  Select,
+  Stack,
+  Table,
+  TableBody,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Tooltip,
+  Typography
+} from "@mui/material";
+import React, { useEffect, useState } from "react";
+
+type AppPreviewProps = {
+  showPreview: boolean;
+  onClose: () => void;
+};
+
+export const AppPreview = ({ showPreview, onClose }: AppPreviewProps) => {
+  const [isPreviewOpen, setIsPreviewOpen] = useState<boolean>(showPreview);
+
+  /** Handle close `App Preview` dialog */
+  const handleClose = () => {
+    onClose();
+  };
+
+  useEffect(() => {
+    setIsPreviewOpen(showPreview);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [showPreview]);
+
+  return (
+    <>
+      <Dialog open={isPreviewOpen} onClose={handleClose} disableScrollLock>
+        {/* TODO Struttura stack col con 3 box come row */}
+        <Stack
+          flexDirection={"column"}
+          paddingX={3}
+          paddingY={3}
+          sx={{ minWidth: 600, minHeight: 640 }}
+          padding={"32px"}
+          rowGap={"24px"}
+        >
+          <Stack direction={"row"} gap={1}>
+            <Typography fontWeight={700} variant="h6">
+              Anteprima In App IO
```

Comment:
Please add locale string translated in both supported languages (IT and EN)

Reply:
[2e63ea3](https://github.com/pagopa/io-services-cms/pull/810/commits/2e63ea3169b8c3bed36c75069eec8b34c6a1e169)